wagtail/management/commands/purge_revisions.py

```diff
@@ -39,14 +33,14 @@ def handle(self, *args, **options):
 
 def purge_revisions(days=None):
     # exclude revisions which have been submitted for moderation in the old system
-    purgeable_revisions = Revision.page_revisions.exclude(
+    purgeable_revisions = Revision.objects.exclude(
```

Comment:
Should we support purging just page, or just non-page revisions?

Reply:
I was thinking that as well, but didn't want to think about what the option should look like 😆 

The `publish_scheduled` command (which was renamed from `publish_scheduled_pages` command) doesn't support it, though I think it makes sense for that management command as it's more like a setup-and-forget type of command.

It makes sense to support it here, so if we want to, I'm thinking something like:
- `--only=page` for page revisions only
- `--only=other` for non-page revisions only
- unspecified (default), for all revisions

Not sure about `other` vs `snippet` or some other term. I'd prefer not using the `snippet` term in case we make it even more generic in the future, or in case people have `RevisionMixin`-enabled model that's not registered as a snippet.

Suggestions welcome!

Reply:
what about:
- default = all
- `--pages` for page revisions
- `--non-pages` for non-page revisions
?

Reply:
Sounds good, added via `1444fbf` and `423df97`